3001-4000/LIT3326Melchiz1.xml

```diff
@@ -30,17 +35,57 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                   <citedRange unit="item">5280</citedRange>
                </bibl>
             </listBibl>
+            <listWit>
+               <witness corresp="PetermannIINachtrag51">Berlin, Berlin State Library, Petermann II
+                  Nachtrag 51 (ms. B in Weischer's edition)</witness>
+               <witness corresp="Tanasee134">Samen Gondar, Dāgā ʾƎsṭifānos, Ṭānāsee 134 (ms. H in
+                  Weischer's edition)</witness>
+               <witness corresp="Tanasee37">Ṭānā, Kǝbrān Gabrǝʾel, Ṭānāsee 37 (ms. J in Weischer's
+                  edition)</witness>
+               <witness corresp="BLorient740">London, British Library, BL Oriental 740 (ms. K in
+                  Weischer's edition)</witness>
+               <witness corresp="BNFet146">Paris, Bibliothèque nationale de France, BnF Éthiopien
+                  146 (ms. P in Weischer's edition)</witness>
+               <witness corresp="BLorient741">London, British Library, BL Oriental 741 (ms. Q in
+                  Weischer's edition)</witness>
+               <witness corresp="BNFet48">Paris, Bibliothèque nationale de France, BnF Éthiopien 48
+                  (ms. R in Weischer's edition)</witness>
+               <witness corresp="BLorient745">London, British Library, BL Oriental 745 (ms. S in
+                  Weischer's edition)</witness>
+               <witness corresp="UBTaeth28">Tübingen, Universitätsbibliothek Tübingen, Tübingen
+                  Aeth. 28 (ms. T in Weischer's edition)</witness>
+               <witness corresp="IVorlov4">Saint Petersburg, Institut Vostočnyh Rukopisej Rossijskoj
+                  Akademii Nauk, IV Orlov 4 (ms. V in Weischer's edition)</witness>
+               <witness corresp="IVkoriander3">Saint Petersburg, Institut Vostočnyh Rukopisej
+                  Rossijskoj Akademii Nauk, IV Koriander 3 (ms. W in Weischer's edition)</witness>
+               <witness corresp="Tanasee42">Ṭānā, Kǝbrān Gabrǝʾel, Ṭānāsee 42 (ms. Y in Weischer's
+                  edition)</witness>
+               <witness corresp="BLEthiopic3" xml:id="blethiopic3"/>
```

Comment:
@nafisa-valieva I noticed just now I do not think we have any record with xml:id `BLEthiopic3` - what is this witness?

Reply:
Sorry it is there, the online version has been reexpanded now and I can see all is good, forget it